packages/rpc-methods/src/restricted/manageAccounts.ts

```diff
@@ -0,0 +1,129 @@
+import {
```

Comment:
Can we add manifest validation for this RPC method? Should just be adding it here: https://github.com/MetaMask/snaps/blob/main/packages/snaps-utils/src/manifest/validation.ts#L167

Reply:
Done in [5375fcb](https://github.com/MetaMask/snaps/pull/1290/commits/5375fcb116a942f7cdb54936e5a47816a6fcee18)